[2023-08-28T08:21:08.573+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509
[2023-08-28T08:21:08.574+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task'
[2023-08-28T08:21:08.617+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509
[2023-08-28T08:21:08.617+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task'
[2023-08-28T08:21:08.757+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_python_with_xcom_eg1.python_xcom_pull_task scheduled__2023-08-27T06:30:00+00:00 [queued]>
[2023-08-28T08:21:08.766+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_python_with_xcom_eg1.python_xcom_pull_task scheduled__2023-08-27T06:30:00+00:00 [queued]>
[2023-08-28T08:21:08.767+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-28T08:21:08.801+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): python_xcom_pull_task> on 2023-08-27 06:30:00+00:00
[2023-08-28T08:21:08.805+0000] {standard_task_runner.py:57} INFO - Started process 125 to run task
[2023-08-28T08:21:08.808+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_python_with_xcom_eg1', 'python_xcom_pull_task', 'scheduled__2023-08-27T06:30:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/dags_python_with_xcom_eg1.py', '--cfg-path', '/tmp/tmpfb3oiq4l']
[2023-08-28T08:21:08.810+0000] {standard_task_runner.py:85} INFO - Job 90: Subtask python_xcom_pull_task
[2023-08-28T08:21:08.885+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509
[2023-08-28T08:21:08.886+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_xcom_eg1/run_id=scheduled__2023-08-27T06:30:00+00:00/task_id=python_xcom_pull_task'
[2023-08-28T08:21:08.888+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_python_with_xcom_eg1.python_xcom_pull_task scheduled__2023-08-27T06:30:00+00:00 [running]> on host 07fcf706bef2
[2023-08-28T08:21:09.002+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_python_with_xcom_eg1' AIRFLOW_CTX_TASK_ID='python_xcom_pull_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T06:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T06:30:00+00:00'
[2023-08-28T08:21:09.026+0000] {logging_mixin.py:150} INFO - value_2
[2023-08-28T08:21:09.027+0000] {logging_mixin.py:150} INFO - ['알루', '코딩', 'Allu', 'Coding']
[2023-08-28T08:21:09.027+0000] {logging_mixin.py:150} INFO - ['정엽', 'letzgorats', 'Owen', 'Coding']
[2023-08-28T08:21:09.028+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-28T08:21:09.037+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_python_with_xcom_eg1, task_id=python_xcom_pull_task, execution_date=20230827T063000, start_date=20230828T082108, end_date=20230828T082109
[2023-08-28T08:21:09.100+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-28T08:21:09.141+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
